backend/src/v4/models/user.js

```diff
@@ -496,9 +496,10 @@ User.createUser = async function (username, password, customData, tokenExpiryTim
 	};
 
 	cleanedCustomData.billing = await UserBilling.changeBillingAddress(cleanedCustomData.billing || {}, billingInfo);
+	const defaultRole = { db: "admin", role: C.DEFAULT_USER_ROLE };
```

Comment:
might be better to store the whole object as the default role instead of we always seems to use it in this context

call it `defaultRoleObj` or something if you need to make it obvious it's an object

Reply:
you mean like this 

![image](https://user-images.githubusercontent.com/71635617/166914404-4b8ff9cd-3fb1-4d05-b9ba-1076bfa53f54.png)